app/uk/gov/hmrc/gform/scheduler/quartz/jobs/ReNotifyJob.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.scheduler.quartz.jobs
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import com.typesafe.akka.extension.quartz.QuartzSchedulerExtension
+import play.api.inject.ApplicationLifecycle
+import uk.gov.hmrc.gform.scheduler.quartz.{ QScheduledJob, QSchedulingActor }
+import uk.gov.hmrc.gform.scheduler.quartz.QSchedulingActor.ReNotify
+import uk.gov.hmrc.gform.sdes.renotify.ReNotifyService
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ReNotifyJob(
+  val reNotifyService: ReNotifyService,
+  val applicationLifecycle: ApplicationLifecycle,
+  val jobActorSystem: ActorSystem,
+  val jobEnabled: Boolean,
+  val jobExpression: String
+)(implicit ex: ExecutionContext)
+    extends QScheduledJob {
+
+  override val jobName: String = "ReNotifyJob"
```

Comment:
I think we should name a more specific name for this job name, maybe like SDES-ReNotifyJob...

Reply:
> I think we should name a more specific name for this job name, maybe like SDES-ReNotifyJob...
I renamed all renotifications 

renamed all renotifications to sdesrenotif....

I